src/itest/kotlin/nl/lifely/zac/itest/AanvullendeInformatieTaskListenerTest.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package nl.lifely.zac.itest
+
+import io.github.oshai.kotlinlogging.KotlinLogging
+import io.kotest.assertions.json.shouldBeJsonObject
+import io.kotest.assertions.json.shouldContainJsonKeyValue
+import io.kotest.core.spec.Order
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import nl.lifely.zac.itest.client.ItestHttpClient
+import nl.lifely.zac.itest.client.ZacClient
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_SPEC_ORDER_AFTER_SEARCH
+import nl.lifely.zac.itest.config.ItestConfiguration.ZAC_API_URI
+import nl.lifely.zac.itest.config.ItestConfiguration.zaakProductaanvraag1Uuid
+import org.json.JSONArray
+
+/**
+ * This test assumes a human task plan item (=task) has been started for a zaak in a previously run test.
+ */
+@Order(TEST_SPEC_ORDER_AFTER_SEARCH)
+class AanvullendeInformatieTaskListenerTest : BehaviorSpec({
```

Comment:
It's a very nice test but as an integration test it does not really test the `AanvullendeInformatieTaskListener` I would say. The itests treat ZAC as a black box. All they should know about are the ZAC endpoints. I would keep the itest naming that way too. The itests just test ZAC endpoints. Whether there is a listener in the backend doing stuff or not is completely irrelevant to the itest.

Reply:
Renamed to `AanvullendeInformatieTaskCompleteTest`